.github/workflows/superset-e2e.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       # https://github.com/cypress-io/github-action/issues/48
       fail-fast: false
       matrix:
-        parallel_id: [0, 1, 2, 3, 4, 5]
+        parallel_id: [0, 1, 2, 0, 1, 2]
```

Comment:
I don't think you meant to get the changes in this file merged (?)

Reply:
This was a quick test - I skimmed docs and realized parallelism needs to be adjusted in a different manner. I'm removing these changes and will attempt to fix the cypress issues in a separate PR.